### PR TITLE
Show that a Post/Comment has been edited

### LINKED ISSUE #1669: 🚀 [Feature] Show "Edited" on comments and Posts
## :rocket: Feature

A user has pointed out that even if we show placeholders for deleted comments people could still distort replies to a post or a comment by editing the content of a post or comment.

As a first step, we should compare `createdAt` with `updatedAt` and show a hint "Edited" along with the date of the edit. That way, we give readers at least the option to see this, even if they have to look carefully to spot it.

By the way, Github shows a full edit history for each comment.

### Design & Layout

Screenshot - 2019-09-23T004325 311

### Additional context
Here is the post where I got the inspiration: https://alpha.human-connection.org/post/54530e00-032a-4221-a225-9d9caf2a5d3c/welchen-sinn

### PR DESCRIPTION
**Pull request by mattwr18**
### Issues

- fixes #1669

### CODE REVIEW

File: webapp/components/Comment/Comment.vue [hunk 1]
```diff
@@ -199,4 +212,8 @@ span.show-more-or-less {
   margin: 0px 20px;
   cursor: pointer;
 }
+
+.ds-text.italics {
+  font-style: italic;
```
[the remark this answers is not in the document]
Comment (mattwr18) [reply webapp/components/Comment/Comment.vue]: yeah, I was thinking of adding it

File: webapp/components/User/User.vue [hunk 2]
```diff
@@ -11,17 +11,17 @@
...
           <div>
-            <ds-text align="left">
+            <ds-text>
               <b class="username">{{ userName | truncate(18) }}</b>
               <ds-text v-if="dateTime" size="small" color="soft">
                 <ds-icon name="clock" />
                 <client-only>
                   <hc-relative-date-time :date-time="dateTime" />
                 </client-only>
+                <slot name="dateTime"></slot>
```
Comment (mattwr18) [inline webapp/components/User/User.vue]: why is this slot called `dateTime`??

File: webapp/pages/post/_id/_slug/index.vue [hunk 3]
```diff
@@ -6,8 +6,11 @@
       :class="{ 'post-card': true, 'disabled-content': post.disabled }"
     >
       <ds-space margin-bottom="small" />
-      <hc-user :user="post.author" :date-time="post.createdAt" />
-      <!-- Content Menu (can open Modals) -->
+      <hc-user :user="post.author" :date-time="post.createdAt">
+        <template v-slot:dateTime>
```
Comment (mattwr18) [inline webapp/pages/post/_id/_slug/index.vue]: same as above :point_up_2:

File: webapp/components/User/User.vue (hunk 2 above)
Comment (roschaefer) [reply webapp/components/User/User.vue]: My thought was to add additional data to dateTime. Therefore I called the named slot `dateTime`.

**Pull request merged**